replica-common/src/main/java/com/graphhopper/OsmHelper.java

```diff
@@ -1,18 +1,41 @@
 package com.graphhopper;
 
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
 import com.graphhopper.reader.ReaderElement;
+import com.graphhopper.reader.ReaderRelation;
 import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.reader.osm.OSMInput;
+import com.graphhopper.reader.osm.OSMInputFile;
 import com.graphhopper.storage.DataAccess;
 import com.graphhopper.util.BitUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.Map;
+import java.io.File;
+import java.util.*;
 
 public class OsmHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(OsmHelper.class);
+
     private DataAccess nodeMapping;
     private DataAccess artificialIdToOsmNodeIdMapping;
     private DataAccess ghEdgeIdToSegmentIndexMapping;
     private BitUtil bitUtil;
 
+    public static final String OSM_NAME_TAG = "name";
+    public static final String OSM_HIGHWAY_TAG = "highway";
+    public static final String OSM_LANES_TAG = "lanes";
+    public static final String OSM_FORWARD_LANES_TAG = "lanes:forward";
+    public static final String OSM_BACKWARD_LANES_TAG = "lanes:backward";
+
+    // Tags we consider when calculating the value of the `lanes` column
+    public static final Set<String> LANE_TAGS = Sets.newHashSet(OSM_LANES_TAG, OSM_FORWARD_LANES_TAG, OSM_BACKWARD_LANES_TAG);
```

Comment:
nit: i like using immutable types for constant collections like this so they're truly constant, but it's mostly a style thing. up to you!

Reply:
Good call, didn't think of this but it's nice going beyond just `final`; updated!